.github/workflows/node-ci.yml

```
@@ -295,3 +319,17 @@ jobs:
                       Oh noes!! We couldn't find any changesets in this PR (${{
                       steps.short-sha.outputs.short_sha }}). As a result, we did not
                       publish an npm snapshot for you.
+
+            - name: Create or update npm snapshot comment - failure, concurrent with release
+              if: always() && steps.publish-snapshot.outputs.npm_snapshot_tag == '' && steps.check-release.outputs.release-count != '0'
```

🤔  Won't this skip in cases where we didn't get a snapshot tag but the `release-count` is `0`?

Wouldn't this be what we want?

```suggestion
              if: failure() || steps.publish-snapshot.outputs.npm_snapshot_tag == '' || steps.check-release.outputs.release-count != '0'
```

---

Yeah, that's what just happened in my test. 🤦

And yes, we can do `failure()` too :)

---

Hmm, this still didn't work...I must've messed something up here

---

Oh, it was the or vs and 🤦🏻

---

> 🤔 Won't this skip in cases where we didn't get a snapshot tag but the release-count is 0?

Yes, because this is solely for when we have an active release. There is no message for if the publish didn't happen and we hit failure. There's no path for that; never was.

---

I just spotted the bug.

`release-count`. It's output as `release_count` in the step. 🤦🏻